extensions/wikia/GlobalNavigation/templates/GlobalNavigation_hubsMenuSections.php

```diff
@@ -3,16 +3,16 @@
 		<div class="hub-menu-columns">
 			<? if ( array_key_exists( 'children', $sections ) ) : ?>
 				<? foreach( $sections['children'] as $category ): ?>
-					<strong><?= $category['text'] ?></strong>
-					<? if ( array_key_exists( 'children', $category ) ) : ?>
-						<ul>
+					<ul>
+						<li><strong><?= $category['text'] ?></strong></li>
```

Comment:
I'm just wondering if adding a wrapper wouldn't be more semantically correct. Now the higher level list item/header becomes the same level as subitems. What do you think about using `<dl>`/`<dt>`/`<dd>`? It's still not perfectly expressing the semantics, but might be a way out.